### ISSUE TITLE
get() function not working in transcrypt

### ISSUE THREAD

**Issue by jvfe** [berinhard/pyp5js#136]
A simple example for extracting a section of an image, similar to the one in the [p5js reference](https://p5js.org/reference/#/p5/get), doesn't work inside of transcrypt:

```py
from pyp5js import *


img = None 

def setup():
    global img
    img = loadImage('foto_qualquer.png')
    

def draw():
    background(img)
    c = get(50, 90)
    fill(c)
    noStroke()
    rect(25, 25, 50, 50)
```

The error raised is `Uncaught TypeError: _P5_INSTANCE.py_get is not a function`. I wasn't able to reproduce the same results in the pyodide demo.

Edit: An alternative using `img.get(50,90)` doesn't work either, raising the same error.

**Comment by berinhard** [comment]
Thanks for opening this issue @jvfe! 

This error has to do with missing transcrypt's `__pragma__` annotation. Take a look at this references and maybe they can help you to find a way through to fix this bug =)

- [StackOverflow post](https://stackoverflow.com/questions/48848059/retrieve-files-with-jquery-and-transcrypt) with the same error message and possible work around;
- Transcrypt's [aliases map](https://www.transcrypt.org/docs/html/special_facilities.html#pragma-alias);
- [Example](https://github.com/berinhard/pyp5js/blob/develop/pyp5js/pyp5js.py#L194) of disabling alias with __pragma__ in `pyp5js`;

Let me know if I can help you with anything else.

**Comment by jvfe** [comment]
Thank you very much for your tips! I managed to implement the basic get() function using the example you gave me, and it worked flawlessly:

```py
def get(*args):
    __pragma__('noalias', 'get')
    p5_get =  _P5_INSTANCE.get(*args)
    __pragma__('alias', 'get', 'py_get')
    return p5_get
```

However, the class methods for get still don't work, e.g. `p5.Image.get()`, I couldn't find an example of how to implement these, can you give some direction on how I would go about implementing them? I'd be more than happy to solve this issue.

**Comment by berinhard** [comment]
@jvfe can you paste a code snippet example?

**Comment by berinhard** [comment]
@jvfe I've fixed this in the PR #139 

Can you test it again to check if it's working for you? I've tested locally and it's working

**Comment by jvfe** [comment]
> @jvfe I've fixed this in the PR #139
> 
> Can you test it again to check if it's working for you? I've tested locally and it's working

Oh you were quicker than me on this one! 🤣

But, yep! I've checked out your PR locally and it's working fine, thank you!
